doc/nrf/protocols/matter/index.rst

```diff
@@ -20,9 +20,7 @@ The |NCS| allows you to develop applications with different versions of Matter,
 +--------------------------+-----------------------------------------------------+------------------------+
 | nRF Connect SDK version  | Matter specification version                        | `Matter SDK version`_  |
 +==========================+=====================================================+========================+
-| v2.4.99 (latest)         | :ref:`1.1.0 <ug_matter_overview_dev_model_support>` | 1.1.0.1                |
-+--------------------------+                                                     |                        |
-| |release|                |                                                     |                        |
+| |release|                | :ref:`1.1.0 <ug_matter_overview_dev_model_support>` | 1.1.0.1                |
```

Comment:
Should v2.4.0 also be added here? I could see minor releases are also added in the table.

Reply:
This is a very good question. I'd believe we need to have 2.4.0 included. @greg-fer, what do you think?

Reply:
Updated the table to include 2.4.0 + patch releases.

Reply:
Discussed offline with @LuDuda and agreed all previous versions should be in the table. Worked with @peknis to add the values.

Reply:
thanks